hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -239,4 +240,31 @@ public void run()
             }
         });
     }
+
+    @Test
+    public void testListNodes() throws IOException {
+        final HazelcastInstance instance1 = Hazelcast.newHazelcastInstance(config);
+        HTTPCommunicator communicator = new HTTPCommunicator(instance1);
+
+        assertTrue(instance1.getLifecycleService().isRunning());
```

Comment:
why are you testing of the node is running? That is not an essential part of the 'listnodes' test. If you don't have much faith if a node is running after the newHazelcastInstance is called; then write explicit tests for that at the appropriate location.